src/fdosecrets/objects/Prompt.cpp

```diff
@@ -303,9 +303,7 @@ namespace FdoSecrets
                     }
                     continue;
                 }
-                // attach a temporary property, so later we can get the item
-                // back from the dialog's result
-                entry->setProperty(FdoSecretsBackend, QVariant::fromValue(item.data()));
+                m_entry2items[entry] = item.data();
```

Comment:
Prefer this to be named `m_entryToItems` and use entry->uuid() instead of a pointer since we don't actually need the entry pointer itself.

Reply:
Never mind, I took care of it.